.github/workflows/subber.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
           COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           COVERALLS_PARALLEL: true
           COVERALLS_FLAG_NAME: 'Subber'
+          COVERALLS_SERVICE_JOB_ID: ${{ github.sha }}
```

Comment:
It looks like `${{ github.sha }}` is the only common reference we can use between different workflows.